icu4c/source/test/Makefile.in

```diff
@@ -32,7 +32,7 @@ STATUS_FULL = $(shell pwd)/$(STATUS_TMP)
 ## Files to remove for 'make clean'
 CLEANFILES = *~ $(STATUS_TMP)
 
-SUBDIRS = $(TESTDATA) intltest $(IOTEST) cintltst $(LETEST)
+SUBDIRS = $(TESTDATA) intltest $(IOTEST) cintltst $(LETEST) fuzzer
```

Comment:
is there any reason the fuzzer should be optional? I.e. disableable from `configure` ?

Reply:
^ yes, I think it should be off by default since with uconfig options, some of the fuzzers can't run.

Reply:
Optional means to put it as $(FUZZER), I assume? with the variable instantiated as needed.

Reply:
right, with a `--enable-fuzzer` configure option (see `--enable-tools`, etc)

Reply:
If the fuzzer build is off by default, we also need to explicitly turn it on for the Clang / Linux continuous build.

Reply:
we might even be able to turn it on for _most_ of our builds.

Reply:
Added enablement setting to configure.ac, re-created configure, modified test/Makefile.in.